internal/application/application.go

```diff
@@ -184,29 +136,8 @@ func WorkingDirectory(app *redskyappsv1alpha1.Application) string {
 	return ""
 }
 
-// LoadResources loads all of the resources for an application, using the supplied file system
-// to load file based resources (if necessary).
-func LoadResources(app *redskyappsv1alpha1.Application, _ filesys.FileSystem) (resmap.ResMap, error) {
-	kf := scan.NewKonjureFilter(WorkingDirectory(app), nil)
-	kf.KeepStatus = false
-
-	var buf bytes.Buffer
-	err := kio.Pipeline{
-		Inputs:  []kio.Reader{app.Resources},
-		Filters: []kio.Filter{kf},
-		Outputs: []kio.Writer{&kio.ByteWriter{Writer: &buf}},
-	}.Execute()
-	if err != nil {
-		return nil, err
-	}
-
-	dp := provider.NewDefaultDepProvider()
-	f := resmap.NewFactory(dp.GetResourceFactory(), dp.GetConflictDetectorFactory())
-	return f.NewResMapFromBytes(buf.Bytes())
-}
-
 func cleanName(n string) string {
```

Comment:
nit, would it make sense to make this a method off Application?

Reply:
Only if the function was called `ValidateCreate`, `ValidateUpdate` and `ValidateDelete` :)

But in all seriousness, that is probably a better representation of what we want here: we never want the names to be invalid to begin with.